polaris/polaris/integrations/javascript.py

```diff
@@ -1,21 +1,40 @@
 from typing import List, Dict, Optional
 
 
-def scripts(page_content: Optional[Dict]) -> List[str]:
+class TemplateScript:
+    def __init__(self, file_path: str = None, url: str = None, is_async: bool = False):
```

Comment:
Can we change `file_path` to `path` and `is_async` to `async`?

Reply:
I can change to path, async is reserved by python so throws an error when you try to use as an argument name. 

I thought is_async would be better than is_sync, since async is the attribute getting added to the html

Reply:
Oh right ok no worries then, `is_async` is alright